modules/test/playwright/tests/object-web/objectEntries.spec.ts

```diff
@@ -124,15 +126,15 @@ test.describe('Manage object entries through Page Templates', () => {
 				},
 			});
 
-		objectDefinitions.push(objectDefinition1);
+		apiHelpers.data.push({id: objectDefinition1.id, type: 'objectDefinition'})
```

Comment:
I guess we need to remove all the tear-down process we are doing in the afterEach method of test..